tests/vectordb/test_lancedb.py

```diff
@@ -0,0 +1,221 @@
+import os
+import shutil
+
+import pytest
+
+from embedchain import App
+from embedchain.config import AppConfig
+from embedchain.config.vectordb.lancedb import LanceDBConfig
+from embedchain.vectordb.lancedb import LanceDB
+
+os.environ["OPENAI_API_KEY"] = "test-api-key"
```

Comment:
And this shouldn't be used here. Please see this as a reference https://github.com/embedchain/embedchain/blob/main/tests/llm/test_openai.py#L12

Reply:
ok, I'll make changes and update